Massonus/HW5/HW5.3.py

```diff
@@ -0,0 +1,8 @@
+number = int(input("Enter a number: "))
```

Comment:
good job
look at this implementation
input_number = int(input("Input natural number: "))

i = 1
fact = 1

if input_number == 1 or input_number == 0:
    print("Factorial {}! =".format(input_number), fact)
elif input_number < 0:
    print("Factorial non positive number don't exist")
else:
    while i <= input_number: 
        fact *= i
        i += 1
        print("Factorial {}! =".format(input_number), fact)